app/code/Magento/Sales/Api/Data/ShipmentTrackInterface.php

```diff
@@ -46,6 +46,10 @@
      */
     const TRACK_NUMBER = 'track_number';
     /*
+     * Track URL.
+     */
+    const TRACK_URL = 'track_url';
```

Comment:
this constant does not have corresponding getter and setter in this interface and should not reside here, rather in the interface which has setter for it.